MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
> SDL does not have a way of setting window resizable parameter after the window has already been created.

We will have to find a way around that.  At least on startup we could delay creation of the SDL window until `Game.Initialize()` which currently when the window is actually shown.

But changing that after startup... including making the window borderless...  is all stuff we need to support.

Regardless of all that we won't be adding a `Game.Resizable` property.